torch_geometric/sampler/neighbor_sampler.py

```diff
@@ -119,11 +119,10 @@ def __init__(
             feature_store, graph_store = data
 
             # Obtain graph metadata:
-            node_attrs = [
-                attr for attr in feature_store.get_all_tensor_attrs()
-                if isinstance(attr.group_name, NodeType)
-            ]
-            self.node_types = list(set(attr.group_name for attr in node_attrs))
+            node_attrs = feature_store.get_all_tensor_attrs()
```

Comment:
Can you clarify for me why this fixes temporal sampling?

Reply:
But `time_attrs` is a subset of `node_attrs` for which group names are a node type. If the `node_attrs` are empty here, I would expected `time_attrs` to be empty as well?

Reply:
It is about the condition that checks whether the attr.group_name is of NodeType. In the case of homo graphs, group name will always be None because there are no node types. Therefore, in the case of homo graphs, node_attrs will always be empty.

Reply:
It is needed because for some reason get_all_tensor_attrs returns edge_types as well.

Reply:
I'm sorry but I need to go. I'll check that tomorrow.

Reply:
Ok, that makes sense to me.